lib/epicenter_web/live/import_live.html.heex

```diff
@@ -0,0 +1,17 @@
+<section id="import-page">
+  <h2>Import Labs</h2>
+  <%= if Euclid.Exists.present?(@import_error_message) do %>
```

Comment:
I believe this will be extracted into a core component in the future.

Reply:
added https://github.com/RatioPBC/epi-viewpoint/issues/14